debian/snapd.postinst

```diff
@@ -11,4 +11,10 @@ case "$1" in
         if dpkg --compare-versions "$2" lt-nl "2.0.7"; then
             ldconfig
         fi
+        # start required services
+        if [ -d /run/systemd/system ]; then
+            systemctl daemon-reload
+            systemctl -f enable snapd.refresh.timer snapd.refresh.service snapd.autoimport.service snapd.socket snapd.service snap.mount.service
```

Comment:
The debhelper generated scripts use deb-systemd-helper which is doing more than this, it also keeps some persistent state. It makes me slightly uneasy that we are not using this.